atomic_reactor/inner.py

```diff
@@ -325,14 +330,28 @@ def build_docker_image(self):
                 self.autorebuild_canceled = True
                 raise
 
+            start_time = datetime.datetime.now()
+            self.plugins_timestamps['dockerbuild'] = start_time.isoformat()
```

Comment:
That's not really correct - there's no way to know server's tz and properly convert it to UTC and indicate it with 'Z'